arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -168,7 +168,7 @@ arangodb::Result getReadLockId (
 
 
 arangodb::Result count(
-  std::shared_ptr<arangodb::LogicalCollection> const col, uint64_t& c) {
+  std::shared_ptr<arangodb::LogicalCollection> const &col, uint64_t& c) {
```

Comment:
Mini-Kleinigkeit: Meiner Ansicht nach gehört das & zum Typ und nicht zum Namen des Arguments. Deswegen schreibe ich normalerweise:

std::shared_ptr<LogicalCollection> const& col

Wir hatten das auch früher mal in den guidelines, keine Ahnung, ob das noch so ist. Auch keine Ahnung, was clang-format daraus macht.